app/views/partials/populate.ejs

```diff
@@ -0,0 +1,63 @@
+<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.1.0/css/font-awesome.min.css"/>
```

Comment:
I think this file shouldn't be in this pull-request, as it's supposed to have only the API stuff, and accessed through Postman... pls remove from this branch